software/common/common.sh

```diff
@@ -110,11 +110,21 @@ install_linux() {
   pip3 install -U pip
   pip3 install codecov
   pip3 install platformio==${PIO_VERSION}
+
+  # upgrading platformio to development version to get proper code coverage reports
+  # remove this when platformio 6.1.0 goes live.
```

Comment:
maybe add a `TODO` here, so it's easy to search for

Reply:
Absolutely